index.html

```diff
@@ -11,22 +11,214 @@
+                   don’t hestiate to contact me.</p>
+                <button class="projects" type="button">Get My Resume</button>
+                </div>
+                <br>
+                <div class="planholder">
+                    <div class="box1">
+                        <div class="skills">
+                            <img src="images/icon-languages.svg" alt="icon languages" class="icon">
+                            <h2 class="skills">Languages</h2>
+                            <ul class="language">
+                            
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">css</button></li>
+                            <li><button type="button">Java Script</button></li>
+                            <li><button type="button">HTML</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box2">
+                        <div class="skills">
+                            <img src="images/icon-frameworks.svg" alt="icon Frameworks" class="icon">
+                            <h2 class="skills">Frameworks</h2>
+                            
+                            <ul class="language">
+                            <li><button type="button">Bootstrap</button></li>
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">RSpec</button></li>
+                            <li><button type="button">Capybara</button></li>
+                            <li><button type="button">Selenium</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box3">
+                        <div class="skills">
+                            <img src="images/icon-skills.svg" alt="icon skills" class="icon">
+                            <h2 class="skills">Skills</h2>
+                            <ul class="language">
+                            <li><button type="button">Codekit</button></li>
+                            <li><button type="button">GitHub</button></li>
+                            <li><button type="button">Codepen</button></li>
+                            <li><button type="button">Gitlab</button></li>
+                            <li><button type="button">Terminal</button></li>
+                        </ul>
+                        </div>        
+                    </div>
+                </div>
+            <br>
+            <br>
+            <br>
+            <div class="aboutyou">
+                <div class="comment"><h2 class="aboutyou-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
```

Comment:
- [ ] (Optional) It would be nice to make the `font-color` for this text the same as that of the Design template. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172944823-b70f50d8-9d74-434b-b709-0c51a4a0c0d4.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172944893-b0384467-9120-4324-ad5d-60ff5ceb02c5.png)